pysd/py_backend/xmile/xmile2py.py

```diff
@@ -75,40 +81,44 @@ def get_xpath_text(element, path, ns={'ns': NS}, default=''):
         doc = get_xpath_text(node, 'ns:doc')
         py_name = namespace[name]
         
-        inflows = [e.text for e in
-                   node.xpath('ns:inflow', namespaces={'ns': NS})]
-        outflows = [e.text for e in
-                    node.xpath('ns:outflow', namespaces={'ns': NS})]
+        # Extract input and output flows equations
+        inflows = [n.text for n in node.xpath('ns:inflow', namespaces={'ns': NS})]
+        outflows = [n.text for n in node.xpath('ns:outflow', namespaces={'ns': NS})]
         
-        eqn = ' + '.join(inflows)
-        eqn += ' - ' + ' - '.join(outflows) if outflows else ''
+        eqn = ' + '.join(inflows) if inflows else ''
```

Comment:
This is a great catch! I guess we should test the case in which there is only an outflow from a stock. That's an xmile specific paradigm, so it hasn't been included in the test suite so far.